amplify/backend/function/postUser/src/index.py

```diff
@@ -10,27 +10,22 @@
 db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
 
 def lambda_handler(event, context):
-    userData = event["userInfo"]
-    firstName = userData["firstName"]
-    lastName = userData["lastName"]
-    dob = userData["DOB"]
-    username = userData["username"]
+    userData = event["request"]["userAttributes"]
+    # Inputs currently required by cognito
+    firstName = userData["given_name"]
+    lastName = userData["family_name"]
+    username = userData["preferred_username"]
     email = userData["email"]
-    gender = userData["gender"]
-    bio = userData["bio"]
+    sub = userData["sub"]
 
-    query = f"g.addV('user').property('firstName', '{firstName}').property('lastName', '{lastName}').property('DOB', '{dob}').property('username', '{username}').property('email', '{email}').property('gender', '{gender}').property('bio', '{bio}');"
-        
-    try:
-        db.submit(query)
-        result = f"User : {username} created successfully"
-        statusCode = 201
-    except:
-        result = f"Server failed at creating user : {username}"
-        statusCode = 500
+    # Inputs not required by cognito
+    dob = ""
+    gender = ""
+    bio = ""
+
+    # Add a user
+    query = f"g.addV('user').property('firstName', '{firstName}').property('lastName', '{lastName}').property('DOB', '{dob}').property('username', '{username}').property('email', '{email}').property('gender', '{gender}').property('bio', '{bio}').property('id', '{sub}');"
+    db.submit(query)
```

Comment:
Remove variables since they are just empty strings and update the comment accordingly
```suggestion
    # Creates a user vertex using attributes provided by Cognito and initializing the
    # remaining attributes to empty strings (users can update these attributes later)
    query = f"g.addV('user').property('firstName', '{firstName}').property('lastName', '{lastName}').property('username', '{username}').property('email', '{email}').property('id', '{sub}').property('DOB', '').property('gender', '').property('bio', '');"
```

Reply:
Gotcha that makes sense